framework/doc/content/syntax/Debug/index.md

```diff
@@ -58,6 +59,23 @@ interact in arbitrarily complex ways on a group of variables are indeed executed
 by the modeler. If problematic, object execution may be reordered using various boolean parameters, `execute_on` flags, and other manual dependency declarations.
 For example for UserObjects, the `execution_order_group` parameter lets the modeler select the ordering of executions of user objects.
 
+## Viewing objects created by an applications
+
+Numerous applications will use [Actions](actions/Action.md) to simplify the user input. This reduces opportunities for
+mistakes in an input, but has the inconvenience of hiding part of the simulation setup. The [!param](/Debug/show_actions) will,
+for most objects, list the objects created by an action. The `Debug` system also offers several summaries of objects:
+
+- [!param](/Debug/show_material_props) for material properties, created on elements, neighbors and sides
+- [!param](/Debug/show_reporters) for reporters, created directly or from systems that derive from Reporters, such as [VectorPostprocessors](syntax/Postprocessors/index.md) and [Positions](syntax/Positions/index.md)
+- [!param](/Debug/show_functors) for [Functors](syntax/Functors/index.md), an abstraction for objects which includes [Functions](syntax/Functions/index.md) and [Variables](syntax/Variables/index.md)
+
+
+Additionally, [!param](/Debug/show_execution_order) will provide the list of objects executed as they are executed.
+This includes, [Kernels](syntax/Kernels/index.md) (and derived classes), [AuxKernels](syntax/AuxKernels/index.md), [boundary conditions](syntax/BCs/index.md)
+(and declinations), [UserObjects](syntax/UserObjects/index.md), [Postprocessors](syntax/Postprocessors/index.md) and
```

Comment:
I think that derived classes is kind of implicit. If it's not, then it seems like it should be consistently applied to all the systems listed here. (There is also one more "declinations")

Reply:
ok I ll reword
they are not derived classes actually, what was I thinking